App/components/shared/Pagination.tsx

```diff
@@ -46,13 +46,17 @@ export const Pagination: React.FC<IPaginationProps> = ({
     }
   }
 
+  // The `TouchableOpacity` prop `disabled` is commented out below because it
```

Comment:
Will there ever be the need to disable the back button in lieu of the comment? If not then we can just remove the property along with the note.

Reply:
There seems to be a bug where, if I disable a control then enable it the accessibility label does not show. My work around is to just change the colour to transparent - which I don't love. It causes the button to flicker on my phone. It's better to disable it. I was going to update RN in a later release and monitor to see if the problem goes away. The comment is more of a reminder for me to monitor the bug.

Reply:
PS: I can remove it in favour of a GitHub issue.

Reply:
Let's track it in a GH issue. For now just remove the disabled prop altogether and update the `FIXME` comment so devs know that it shouldn't be added.

Reply:
Fixed. Issue #204 created to track.